File: Lec -1 C Programming/scores.c
#include <cs50.c>
#include <stdio.h>

int main(void) {
    int n = get_int("How many numbers: ");
    int scores[n];
    for (int i = 0; i < n; i++) {
        scores[i] = get_int("Score: ");
    }

    int sum = 0;
    for (int i = 0; i < n; i++) {
        sum += scores[i];
    }

    printf("Average Score: %.2f\n", (float)sum / (float)n);
}

File: Lec - 6 Python/scores.py
import cs50

scores = []

for i in range(3):
    score = cs50.get_int("Scores : ")
    scores.append(score)
    #scores += [score]
    #scores = scores + [score]
average = sum(scores)/len(scores)
print ("Average : " + str(average))
                         